cypress/e2e/EngageSphere_A11Y.cy.js

```diff
@@ -0,0 +1,27 @@
+//Tabela de clientes
+
+import 'cypress-axe';
+describe('Acessibilidade da Tabela de Clientes', () => {
+    beforeEach(() => {
+      cy.visit('/');
+      cy.injectAxe();
+    });
+
+    it('não encontra problemas de acessibilidade no modo claro', () => {
+      cy.document().then((doc) => {
+        doc.body.setAttribute('data-theme', 'light');
+      });
+      cy.get('[aria-label="theme-toggle-button"]').should('contain', '☽');
+      cy.get('[data-testid="table"]', { timeout: 10000 }).should('be.visible');
+      cy.checkA11y()
+    });
+
+    it('não encontra problemas de acessibilidade no modo escuro', () => {
+
+      cy.get('#theme-toggle-button').click();
+      cy.get('[aria-label="theme-toggle-button"]').should('contain', '☀');
+      cy.get('[data-testid="table"]', { timeout: 10000 }).should('be.visible');
+      cy.checkA11y()
+
+    });
+  });
```

Comment:
No entanto, ao final do arquivo é recomendado deixar uma linha em branco, para evitar e exibição do ícone com o `aria-label="No newline at end of file"` no GitHub.

```suggestion
  });
  
```